.github/workflows/main.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      max-parallel: 3
```

Comment:
curious on why we're making this change?

Reply:
CI was unstable. Removed all limitation. Works ok